common/arch/win32/rbaudio.cpp

```diff
@@ -36,7 +37,8 @@ namespace dcx {
 
 static UINT wCDDeviceID = 0U;
 static int initialised = 0;
-static DWORD playEnd;
+static int playEnd = 0;
```

Comment:
`static` variables automatically initialize to `0`.

Reply:
`int` is signed, whereas `DWORD` is unsigned.  Why do you need to change the signedness of this variable?

Reply:
The value can actually never be negative, and `playEnd` is always assigned from `dwReturn`, itself `DWORD_PTR` but actually containing a `DWORD` value.

Reply:
On Win64, `DWORD_PTR` will be 64 bits, but `DWORD` will be 32 bits.  Will that truncation cause an issue?

Reply:
MSF is a 32-bit format as specified by the MCI documentation, so no, truncation won't be an issue, but I'll add a comment about it.